### PR TITLE
Minor documentation update and exception message fix to avoid confusion with Google Vertex

### LINKED ISSUE #2087: Chat exception message contains 'vertex' keyword which is confusing when used with other providers
**Describe the bug**
Here is an example error exception message which contains the word `vertex` which is confusing (similar to google vertex) as we're using Azure openai provider here,
```
ValueError: Error building vertex Azure OpenAI: Error code: 400 - {'error': {'code': 'OperationNotSupported', 'message': 'The chatCompletion operation does not work with               
                             the specified model, text-embedding-ada-002. Please choose different model and try again. You can learn more about which models can be used with each operation here:                   
                             [https://go.microsoft.com/fwlink/?linkid=2197993.'}}](https://go.microsoft.com/fwlink/?linkid=2197993.%27}})
```

**Browser and Version**
n/a

**To Reproduce**
n/a

**Screenshots**
n/a

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by msmygit**
Fixes #2087

### CODE REVIEW

File: src/backend/base/langflow/api/v1/chat.py [hunk 1]
```diff
@@ -179,7 +179,7 @@ async def build_vertex(
             result_data_response = ResultDataResponse(**result_dict.model_dump())
 
         except Exception as exc:
-            logger.exception(f"Error building vertex: {exc}")
+            logger.exception(f"Error building langflow vertex: {exc}")
```
Comment (ogabrielluiz) [inline src/backend/base/langflow/api/v1/chat.py:182]: I'd say we just make it `Error building Component: {exc}`

File: src/backend/base/langflow/api/v1/chat.py [hunk 2]
```diff
@@ -229,7 +229,7 @@ async def build_vertex(
         )
         return build_response
     except Exception as exc:
-        logger.error(f"Error building vertex: {exc}")
+        logger.error(f"Error building langflow vertex: {exc}")
```
Comment (ogabrielluiz) [inline src/backend/base/langflow/api/v1/chat.py:232]: Same here

File: src/backend/base/langflow/api/v1/chat.py [hunk 3]
```diff
@@ -324,7 +324,7 @@ async def stream_vertex():
                     raise ValueError(f"No result found for vertex {vertex_id}")
 
             except Exception as exc:
-                logger.exception(f"Error building vertex: {exc}")
+                logger.exception(f"Error building langflow vertex: {exc}")
```
Comment (ogabrielluiz) [inline src/backend/base/langflow/api/v1/chat.py:327]: Same here

File: src/backend/base/langflow/api/v1/chat.py [hunk 4]
```diff
@@ -335,4 +335,4 @@ async def stream_vertex():
 
         return StreamingResponse(stream_vertex(), media_type="text/event-stream")
     except Exception as exc:
-        raise HTTPException(status_code=500, detail="Error building vertex") from exc
+        raise HTTPException(status_code=500, detail="Error building langflow vertex") from exc
```
Comment (ogabrielluiz) [inline src/backend/base/langflow/api/v1/chat.py:338]: And here

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]
Hey @msmygit 

How are you? 

Thank you for the PR. 

Just fix these comments and this should good to go.

File: src/backend/base/langflow/api/v1/chat.py (hunk 4 above)
Comment (msmygit) [reply src/backend/base/langflow/api/v1/chat.py]: done

File: src/backend/base/langflow/api/v1/chat.py (hunk 3 above)
Comment (msmygit) [reply src/backend/base/langflow/api/v1/chat.py]: done

File: src/backend/base/langflow/api/v1/chat.py (hunk 2 above)
Comment (msmygit) [reply src/backend/base/langflow/api/v1/chat.py]: done

File: src/backend/base/langflow/api/v1/chat.py (hunk 1 above)
Comment (msmygit) [reply src/backend/base/langflow/api/v1/chat.py]: done

**Review by ogabrielluiz — APPROVED** [review]
LGTM 

Thanks!

**Pull request merged**